cluster/gce/gci/mounter/mounter.go

```diff
@@ -68,7 +68,7 @@ func mountInChroot(rootfsPath string, args []string) error {
 	args = append([]string{rootfsPath, mountCmd}, args...)
 	output, err := exec.Command(chrootCmd, args...).CombinedOutput()
 	if err == nil {
-		return err
+		return nil
```

Comment:
Thank you for reviewing. I think  "return nil' is correct. That means command executed successfully. So it has no error and returns.